test/selector_assertions_test.rb

```diff
@@ -210,6 +210,20 @@ def test_assert_select_text_match
     end
   end
 
+  def test_assert_select_with_invalid_range
+    render_html "<div>foo</div>"
+    error = assert_raises(ArgumentError) { assert_select("div", 2..1) { nil } }
+    assert_equal "Range begin or :minimum cannot be greater than Range end or :maximum", error.message
+  end
+
+  def test_assert_select_with_invalid_minimum_and_maximum
+    render_html "<div>foo</div>"
+    error = assert_raises(ArgumentError) { assert_select("div", maximum: 0) { nil } }
+    assert_equal "Range begin or :minimum cannot be greater than Range end or :maximum", error.message
```

Comment:
This case might be confusing to a user cause what it's really saying is "we default to a minimum of 1 [here](https://github.com/rails/rails-dom-testing/blob/5fd54c2b86e7a85a87e7178fb77b150d58bd6f42/lib/rails/dom/testing/assertions/selector_assertions/html_selector.rb#L132) and the provided maximum is less than that".

I debated adding a separate message just for this (`"Cannot provide 0 as :maximum without providing :minimum"`) but didn't end up doing so cause this is probably be a very rare case.

Reply:
I think it's fine as it is.